src/handlers/notify_zulip.rs

```diff
@@ -21,23 +21,30 @@ pub(super) fn parse_input(
     if let IssuesAction::Labeled | IssuesAction::Unlabeled = event.action {
         let applied_label = &event.label.as_ref().expect("label").name;
         if let Some(config) = config.and_then(|c| c.labels.get(applied_label)) {
-            for label in &config.required_labels {
-                let pattern = match glob::Pattern::new(label) {
-                    Ok(pattern) => pattern,
-                    Err(err) => {
-                        log::error!("Invalid glob pattern: {}", err);
-                        continue;
+            let contains_label = |haystack: &[Label], needles: &[String]| {
+                for needle in needles {
+                    let pattern = match glob::Pattern::new(needle) {
+                        Ok(pattern) => pattern,
+                        Err(err) => {
+                            log::error!("Invalid glob pattern: {}", err);
+                            continue;
+                        }
+                    };
+                    if haystack.iter().any(|l| pattern.matches(&l.name)) {
+                        return true;
                     }
-                };
-                if !event
-                    .issue
-                    .labels()
-                    .iter()
-                    .any(|l| pattern.matches(&l.name))
-                {
-                    // Issue misses a required label, ignore this event
-                    return Ok(None);
                 }
+                false
+            };
+
+            if !contains_label(event.issue.labels(), &config.required_labels) {
+                // Issue misses a required label, ignore this event
```

Comment:
This comment sounds a little misleading - I think the behavior is that if the issue is missing *all* of the required labels, then we will ignore this event. If it's just missing one of the patterns, then we will not early return here. Right?